examples/quick-start-typescript/docker-compose.yaml

```diff
@@ -0,0 +1,168 @@
+      - COLLECTOR_ZIPKIN_HOST_PORT=:9411
+      - COLLECTOR_OTLP_ENABLED=true
+    healthcheck:
+      test: ['CMD', 'wget', '--spider', 'localhost:16686']
+      interval: 1s
+      timeout: 3s
+      retries: 60
+
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.59.0
+    restart: unless-stopped
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
+    ports:
+      - 55679:55679
+      - 8888:8888
+      - 4317:4317
+      - 4318:4318
+    command:
+      - '--config'
+      - '/otel-local-config.yaml'
+    volumes:
+      - ./collector.config.yaml:/otel-local-config.yaml
+    environment:
+      - JAEGER_ENDPOINT=jaeger:14250
+    depends_on:
+      jaeger:
+        condition: service_healthy
+
+  api:
+    build: .
+    environment:
+      REDIS_URL: cache
+      DATABASE_URL: postgresql://ashketchum:squirtle123@db:5432/pokeshop?schema=public
+      RABBITMQ_HOST: queue
+      POKE_API_BASE_URL: https://pokeapi.co/api/v2
+      COLLECTOR_ENDPOINT: http://otel-collector:4317
+      ZIPKIN_URL: http://localhost:9411
+    healthcheck:
+      test: ['CMD', 'wget', '--spider', 'localhost:8081/pokemon/healthcheck']
+      interval: 1s
+      timeout: 3s
+      retries: 60
+    ports:
+      - 8081:8081
+    depends_on:
+      db:
+        condition: service_healthy
+      cache:
+        condition: service_healthy
+      queue:
+        condition: service_healthy
+      jaeger:
+        condition: service_healthy
+      otel-collector:
+        condition: service_started
+  rpc:
+    build: .
+    environment:
+      REDIS_URL: cache
+      DATABASE_URL: postgresql://ashketchum:squirtle123@db:5432/pokeshop?schema=public
+      RABBITMQ_HOST: queue
+      POKE_API_BASE_URL: https://pokeapi.co/api/v2
+      COLLECTOR_ENDPOINT: http://otel-collector:4317
+      ZIPKIN_URL: http://localhost:9411
+      NPM_RUN_COMMAND: rpc
+    healthcheck:
+      test: ['CMD', 'wget', '--spider', 'localhost:8081/pokemon/healthcheck']
+      interval: 1s
+      timeout: 3s
+      retries: 60
+    ports:
+      - 8082:8082
+    depends_on:
+      db:
+        condition: service_healthy
+      cache:
+        condition: service_healthy
+      queue:
+        condition: service_healthy
+      jaeger:
+        condition: service_healthy
+      otel-collector:
+        condition: service_started
+
+  worker:
+    build: .
+    environment:
+      REDIS_URL: cache
+      DATABASE_URL: postgresql://ashketchum:squirtle123@db:5432/pokeshop?schema=public
+      RABBITMQ_HOST: queue
+      POKE_API_BASE_URL: https://pokeapi.co/api/v2
+      COLLECTOR_ENDPOINT: http://otel-collector:4317
+      ZIPKIN_URL: http://localhost:9411
+      NPM_RUN_COMMAND: worker
+    depends_on:
+      db:
+        condition: service_healthy
+      cache:
+        condition: service_healthy
+      queue:
+        condition: service_healthy
+      jaeger:
+        condition: service_healthy
+      otel-collector:
+        condition: service_started
+
+  tracetest-agent:
```

Comment:
Putting this at the top with `tracetest-client` would make sense to keep them together for clarity.